### PR TITLE
Add required box to Geolocation and disableDragDrop on Info [INTEGRALCS-13751]

### PR DESCRIPTION
**Pull request by paulzala**
Purpose: Add required box to Geolocation and disableDragDrop on Info.
Encountered some repo setup issues, so added some documentation in the hopes that it may help.

### CODE REVIEW

File: src/scripts/fields/info.coffee [hunk 1]
```diff
@@ -36,6 +36,7 @@ Formbuilder.registerField 'info',
     $('.fb-info-editor').summernote(
         onChange: -> update.call(@)
         onKeyup: -> update.call(@)
+        disableDragAndDrop: true
```
Comment (raycui2011) [inline src/scripts/fields/info.coffee:39]: Any reason why we need disable drag and drop?

File: src/scripts/fields/geoLocation.coffee [hunk 2]
```diff
@@ -4,27 +4,14 @@ Formbuilder.registerField 'geolocation',
...
-    <label class='section-name'><%- rf.get(Formbuilder.options.mappings.LABEL) %></label>
     <div class="form-group">
      <button class='fb-button'>Get GeoLocation</button>
     </div>
-    <p><%- rf.get(Formbuilder.options.mappings.DESCRIPTION) %></p>
-
     <span><%=rf.geolocationFunctionality%></span>
   """
 
   edit: """
```
Comment (raycui2011) [inline src/scripts/fields/geoLocation.coffee:14]: Do we need remove the description from geolocation?

File: src/scripts/fields/info.coffee (hunk 1 above)
Comment (paulzala) [reply src/scripts/fields/info.coffee:39]: Thanks Ray - this is actually a carry-over from ticket https://integralcs.atlassian.net/browse/INTEGRALCS-12091 .
When I did 12091, I didn't realise the formbuilder file was generated in another project. I added this line to the integral formbuilder.js file directly.
So whilee I was here, I needed to make sure my changes were updated in this repo to match it. Sorry I missed this the first time. I tested and confirmed that after compiling with grunt that the change to the formbuilder.js file is exactly the same as what I added to integral.

File: src/scripts/fields/geoLocation.coffee (hunk 2 above)
Comment (paulzala) [reply src/scripts/fields/geoLocation.coffee:14]: Yes! What I discovered was that when a formbuilder field is given the type non_input, a bunch of automatic styling including the label, description <p>, and the 'required' asterisk are removed. For this reason, the person who implemented the geolocation and gave it a non_input type had to manually add this stuff in. Now that I'm making it a regular input field again, I needed to remove the extra styling, or else the geolocation box ends up with two labels and two descriptions..!

**Review by raycui2011 — APPROVED** [review]

**Pull request merged**